content/frameworks/react/getting-started/index.md

```diff
@@ -0,0 +1,97 @@
+---
+page_type: guide
+title: Getting started
```

Comment:
General comment about Get Started guide...

I like covering why this content is useful, and also showing how to set up a create react app and get going. But I'm not sure if this guide is doing a lot more than that. I kind of wonder if maybe it would be good to reframe the purpose of this get started guide--which is to get them up and running with a react application, that you will then show how to optimize, performance, reliability, etc. You get the idea. Be more direct that you are getting them set up and ready to learn how to optimize a react app.

Reply:
Changed the title to `Get Started: Optimize your React app`. Definitely sounds more clear

Reply:
Tried reframing the intro a bit to sound like that. Only thing is I didn't want to make it sound like "this is the app you would be using to optimize and learn with" as other guides in the section have applications starting at different points.

Let me know if you think it sounds okay now, or if I can tweak something.